finrl/meta/paper_trading/paper_trader.py

```
@@ -46,35 +46,24 @@ def __init__(
-                    # print ( f"state_dict: {agent.act.state_dict()}")
-                    actor.load_state_dict(agent.act.state_dict())
-                    # actor.load_state_dict(agent.act.state_dict())
-                    self.logger.info ( f" actor: {actor} agent: {agent} device: {agent.device}")
-                    self.act = actor
-                    self.device = agent.device
-                except BaseException as e:
-                    raise ValueError(f"Fail to load agent: {e}")
-
-            elif drl_lib == "rllib":
+        if drl_lib == "elegantrl":
+            agent_class = MODELS.get(agent_name)
```

Ruff

Undefined name `agent_name`

(F821)